.env

```diff
@@ -1,3 +1,20 @@
 #### GENERAL CONFIG ####
 # level of logging (1=debug, 2=info, 3=warn, 4=error)
 LOG_LEVEL=2
+
+# location to access mongo database
+MONGO_URL="mongodb://simplemongousername:simplemongopassword@localhost:27030/?authSource=dashboard"
+
+# location of soartech server
+# ST_URL="http://127.0.0.1:8084" # development
+ST_URL="https://darpaitm.caci.com/soartech" # production
```

Comment:
This won't run with the " # production" on the same line it thinks it is part of the string.  You'll need to move the development and production call outs to the lines above the urls.